docs/contributors/contributors-guide.md

```diff
@@ -0,0 +1,171 @@
+3. **Reflect on your experience**: Mention what you learned from working on the PR. For example, how to import images on with React.
+
+#### Highlighting Dev.to Blog Posts
+
+1. **Capture the essence**: Begin with an interesting blog post summary, capturing the main theme or message.
+2. **Discuss the relevance**: Explain why this blog post is important for your open source project or community. Is it about a major update, a community event, or sharing best practices?
+3. **Engage with a call-to-action**: Encourage readers to engage with the post, whether by reading, commenting, or sharing their thoughts on the topic.
+
+If you want more tips on creating effective highlights, check our blog post, ["Choose Your Own Adventure in Open Source: Paths to Success"](https://dev.to/opensauced/choose-your-own-adventure-in-open-source-paths-to-success-167p). 
+
+## Insights: Connecting Your Repositories
+
+OpenSauced Insights you to see the impact of your contributions. You can use insights pages to track repositories' growth, analyze work, and connect with others.
+
+## Why Create an Insights Page
+
+- **Tracking Personal Contributions**: Add repositories that you contributed to and track your contributions.
+- **Building Portfolios and Developing Skills**: Contributing to open source projects can be a great way to develop new skills. Your contributions can also be used as samples for your portfolio. Looking for ways to implement your open source experience in your job search journey? Check out our [job search guide](../job-seekers-guide/introduction.md)! 😄
+- **Identifying Potential Projects to Contribute To**: Finding open source projects to contribute to can be difficult. The Insights feature can help you identify projects to contribute to.
+- **Networking and Community Engagement**: Through the feature's Contributors tab, you can find other people to connect and collaborate with.
+- **Curating Collections for Interest Groups**: You can curate repositories that interest you, for example, top Artificial Intelligence projects.
+
+### Creating a New Insight Page
+
+To connect your repositories, click the "Insights" tab on the dashboard. You will be redirected to a page where you can create your new Insight Page.
+
+There are two ways to add repositories to your Insight Page:
+
+1. **Sync GitHub organization**: Syncing your favorite GitHub organizations is a good idea if you want to keep track of their open source project activities and trends.
+2. **Connect individual repositories**:  Connecting individual repositories is a good idea if you who wants to keep track the contributions of a specific repository or group of repositories. For example, you might want to track of projects you have contributed to or would like to contribute to in the future.
+
+![sync insights gif](../../static/gif/insight-page-demo.gif)
+
+### Using Your Insights Page
```

Comment:
The guides differ from the docs bc they need to go deeper for the audience. So, for example, in this section, each of these should help contributors specifically utilize this information. I suggest asking yourself "Why would this be useful for a contributor?" and then adding the response to each of these sections. 

Another way to look at it is like this: What information could I share with contributors about these features that isn't already in the general documentation?

For example, The repositories tab helps contributors gauge the project's liveliness and community engagement. Active projects are likely to have quicker responses to PRs, issues, and discussions, making them more attractive for ongoing contributions. If a contributor is looking for an active project to contribute to, this is a good way to determine that information. Being able to see who else contributes there is a good way to make a connection to the project or to find support.